src/components/Navbar.jsx

```diff
@@ -1,15 +1,17 @@
 import { NavLink } from 'react-router-dom';
+import user from '../images/user.png';
 
 function Navbar() {
   return (
     <header>
-      <h1>Bookstore</h1>
       <nav>
+        <h1>Bookstore</h1>
         <ul>
-          <li><NavLink to="/" className="navs">Books</NavLink></li>
-          <li><NavLink to="/categories" className="navs">Categories</NavLink></li>
+          <li><NavLink to="/" className="navs">BOOKS</NavLink></li>
+          <li><NavLink to="/categories" className="navs">CATEGORIES</NavLink></li>
```

Comment:
- [ ] Good job in implementing the changes requested by the previous reviewer, However, after a successful run, I noticed that your styling missed some guidelines from the given design, Please match your design to the given requirement, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention on the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Srudent Zep](https://github.com/SnakyMz/Bookstore/assets/43528594/45f9659b-e3ae-45c1-a26d-66b293176fe6)


</td>
      <td>

![BigZep](https://github.com/obibaadoma/book-stores/assets/43528594/f3df91c1-0159-4b9d-ad3e-00d89a12e295)

</td>
    </tr>
  </table>

Reply:
Hello @Alusp , thanks for the review
For the change in the bottom is it the category dropdown or the "ADD BOOK" button
Please reply
thanks